__tests__/AzureSqlAction.test.ts

```diff
@@ -95,5 +149,14 @@ function getInputs(actionType: ActionType) {
                 additionalArguments: '-t 20'
             } as ISqlActionInputs;
         }
+        case ActionType.BuildAndPublish: {
+            return {
+                serverName: 'testServer.database.windows.net',
+                actionType: ActionType.BuildAndPublish,
+                connectionString: new SqlConnectionStringBuilder('Server=testServer.database.windows.net;Initial Catalog=testDB;User Id=testUser;Password=testPassword'),
```

Comment:
Can you please change these to word "placeholder" too

Reply:
Updated